### PR TITLE
explain required Terraform config and init

### PR DESCRIPTION
**Pull request by janto**
Addresses unclear installation instructions https://github.com/cloudflare/cf-terraforming/issues/310 https://github.com/cloudflare/cf-terraforming/issues/328 https://github.com/cloudflare/cf-terraforming/issues/292 https://github.com/cloudflare/cf-terraforming/issues/241

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +115,27 @@ resource "cloudflare_record" "terraform_managed_resource" {
...
+        }	
+    }
+}
+
+provider "cloudflare" {
+# api_token  = ""  ## Commented out as we are using an environment var
+}
+```
+
+You then need to run `terraform init` within this directory to initialize it.
```
Comment (jacobbednarz) [inline README.md:138]: this isn't really the place for telling people _how_ to organise and run Terraform; instead, let's just call out the command explicitly.

```suggestion
* An initialised Terraform directory (`terraform init` has run and providers installed)
```

File: README.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ This tool is ideal if you already have Cloudflare resources defined but want to
 start managing them via Terraform, and don't want to spend the time to manually
 write the Terraform configuration to describe them.
 
+See also the [blog page](https://blog.cloudflare.com/cloudflares-partnership-with-hashicorp-and-bootstrapping-terraform-with-cf-terraforming/) for further details on installation and usage.
```
Comment (jacobbednarz) [inline README.md:15]:
```suggestion
Read the announcement [blog](https://blog.cloudflare.com/cloudflares-partnership-with-hashicorp-and-bootstrapping-terraform-with-cf-terraforming/) for further details on using `cf-terraforming` in your workflow.
```

**Review by jacobbednarz — CHANGES_REQUESTED** [review]
i've left some comments inline for recommended wording changes here as this project shouldn't be telling users how to setup and run the Terraform installation. there is documentation for that and we should lean on that instead.

**Comment by janto** [comment]
I have no big issues with the requested changes. Added.

**Pull request merged**